src/components/Header/index.tsx

```diff
@@ -0,0 +1,9 @@
+type HeaderProps = {
+  headerTitle: string;
+};
+
+const Header = ({ headerTitle }: HeaderProps) => {
+  return <h1 className="text-2xl font-bold mb-8">{headerTitle}</h1>;
```

Comment:
```suggestion
const Header = ({ headerTitle }: HeaderProps) => {
  return <h1 className="HEADER_STYLE.titleText">{headerTitle}</h1>;
```
이런식으로 costants/styles.ts 밑에 빼야하지 않을까요?

Reply:
수정했습니다!🙂